readme.rst

```diff
@@ -293,6 +295,13 @@ The following links can help you with additional configuration:
 - `Sphinx extensions`_
 - `Furo documentation`_ (Furo is the Sphinx theme we use as our base.)
 
+Other resources
+---------------
+
+- `Example product documentation <https://canonical-example-product-documentation.readthedocs-hosted.com/>`_
+- `Example product documentation repository <https://github.com/canonical/example-product-documentation>`_
+- `Starter pack repository <https://github.com/canonical/starter-pack>`_
```

Comment:
This is where we are, so I don't think we need this in here. ;)

Or if we want to make really sure that users find both the repo and the docs, we should have both links. But I think we can do without.